packages/scandipwa/src/component/ProductCard/ProductCard.component.js

```diff
@@ -316,6 +317,7 @@ export class ProductCard extends PureComponent {
             <div
               block="ProductCard"
               elem="Reviews"
+              mods={ { layout } }
```

Comment:
Please make sure that this change is acknowledged.

Reply:
Also, what other kinds of layout can be there?

Reply:
@yeegor Yes, this change is intentional. `ProductCard_Reviews` element requires different styling depending on whether it is `grid` or `list` layout. And different stylings are applying depending on this. Another option is to replace such syntax with for more clarity:
```
const modsOption = layout === 'list' ? 'list' : 'grid';
<div
              block="ProductCard"
              elem="Reviews"
              mods={ { layout:  modsOption} }>
```

Should I change code to smth like that?